### PR TITLE
Adding support for ignoreExitMessage

### LINKED ISSUE #2874: Need to add feature for ignore-exit-message
As discussed here: https://github.com/spring-cloud-task-app-starters/composed-task-runner/issues/61  This feature had a PR sometime ago (before it was migrated to dataflow, but was never merged.   Need to update and migrate it over.

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #2874

### CODE REVIEW

File: spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/TaskLauncherTasklet.java [hunk 1]
```diff
@@ -170,6 +176,10 @@ public RepeatStatus execute(StepContribution contribution,
 			this.executionId = taskOperations.launch(tmpTaskName,
 					this.properties, args);
 
+			Boolean ignoreExitMessage = isIgnoreExitMessage(args, this.properties);
```
Comment (jvalkeal) [inline spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/TaskLauncherTasklet.java:179]: Should we try to use boot's binder to handle parsing a property as this looks quite low level to handle camelCase and kebab-case and args on top of that?

**Comment by cppwfs** [comment]
Updated and rebased

**Comment by jvalkeal** [comment]
Binding values manually like boot does is something like:
https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/blob/main/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/ImagePullPolicy.java#L44
https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/blob/main/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/DeploymentPropertiesResolver.java#L652

**Comment by cppwfs** [comment]
I'm not sure the solution above handles our arguments scenario.

**Comment by cppwfs** [comment]
Updated per code review

**Comment by cppwfs** [comment]
Rebased and signed commit.  Ready for review.

**Comment by jvalkeal** [comment]
Looks good, lets try it.

**Pull request merged**